Tests.Integration.Presentation.Web/Organizations/StsOrganizationSynchronizationApiTest.cs

```diff
@@ -612,6 +752,22 @@ private static void AssertOrgTree(StsOrganizationOrgUnitDTO unit, HashSet<Guid>
             }
         }
 
+        private static void AssertConsequenceLogs(
+            IEnumerable<ConnectionUpdateOrganizationUnitConsequenceDTO> consequences,
+            StsOrganizationChangeLogResponseDTO logs)
+        {
+            foreach (var consequence in consequences)
```

Comment:
when you assert two collections, the first thing to check is if they are of equal size... That stops it before getting to deep into the assertion that would just fail in Assert.NotNull

Also, this current implementation does not catch of the logs collection is larger than the consequences (expected)